prow/plugins/lifecycle/close.go

```diff
@@ -78,20 +92,31 @@ func handleClose(gc closeClient, log *logrus.Entry, e *github.GenericCommentEven
 			break
 		}
 	}
-	if e.IssueAuthor.Login != commentAuthor && !isAssignee {
-		log.Infof("Assigning %s/%s#%d to %s", org, repo, number, commentAuthor)
-		if err := gc.AssignIssue(org, repo, number, []string{commentAuthor}); err != nil {
-			msg := "Assigning you to the issue failed."
-			if ok, merr := gc.IsMember(org, commentAuthor); merr == nil && !ok {
-				msg = "Only kubernetes org members may be assigned issues."
-			} else if merr != nil {
-				log.WithError(merr).Errorf("Failed IsMember(%s, %s)", org, commentAuthor)
-			} else {
-				log.WithError(err).Errorf("Failed AssignIssue(%s, %s, %d, %s)", org, repo, number, commentAuthor)
+	isAuthor := e.IssueAuthor.Login == commentAuthor
+
+	if !isAssignee && !isAuthor {
+		active, err := isActive(gc, org, repo, number)
+		if err != nil {
+			log.Infof("Cannot determine if issue is active: %v", err)
+			active = true // Fail active
+		}
+
+		if active {
+			// Try to assign the issue to the comment author
+			log.Infof("Assign to %s", commentAuthor)
+			if err := gc.AssignIssue(org, repo, number, []string{commentAuthor}); err != nil {
+				msg := "Assigning you to the issue failed."
+				if ok, merr := gc.IsMember(org, commentAuthor); merr == nil && !ok {
+					msg = "Only kubernetes org members may be assigned issues."
```

Comment:
This message shouldn't be Kubernetes specific.

Reply:
Done